Multisig/UI/Transaction/ExecuteTransaction/TransactionExecutionController.swift

```diff
@@ -374,17 +394,19 @@ class TransactionExecutionController {
             return
         }
 
-        guard let key = selectedKey, let keyBalance = key.balance.amount else {
-            return
-        }
+        if remainingRelays <= ReviewExecutionViewController.MIN_RELAY_TXS_LEFT {  // TODO double check if <
```

Comment:
Shouldn't remainingRelays be >?

Reply:
I think it is fone for now. The checks are for the case when the relays are exhausted